.gitignore

```diff
@@ -27,3 +27,10 @@ build/
 
 # Cache for build scripts
 __pycache__
+
+# Files generated for the Android Emulator.
```

Comment:
Is this important just for the 'Cleanup'-stage? If yes, that would not make a difference, as Jenkins will call a 'git clean -fdx' ('Clean before checkout'-option) before running the job, so those files would be removed anyhow. Which removes the need to cover requirements like: resetting the emulator state, reconfigure the emulator if the emulator config is changed, ...
Is it intended for the developer machine? Then it would be fine I think (can find pro's and con's for that).

Reply:
I missed the "Clean before checkout" setting.
So apparently the emulator is always recreated.
I'll remove this part.